hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -211,7 +211,7 @@ <h2 class="formH2" id="faq">Application FAQs</h2>
                 <p class="faqAnswer">Please send an email to <a href="mailto:{{ contact_email }}" class="primaryText hoverLink">{{ contact_email }}</a> with your name, email you used to apply, expected arrival time, and reason as to why you’re arriving late and we’ll save your spot for you.</p>
 
                 <p class="faqQuestion">How does the waitlist work the day of?</p>
-                <p class="faqAnswer">The rank on the waitlist is first-come first serve, meaning the sooner you arrive at the event on Saturday, the higher your placement on our waitlist. At 10AM, if we have any spots remaining, we will be letting waitlisted people in.</p>
+                <p class="faqAnswer">The rank on the waitlist is first-come first serve, meaning the sooner you arrive at the event on {{ event_start_date.strftime("%A") }}, the higher your placement on our waitlist. At {{ event_start_date.strftime("%I%p") }}, if we have any spots remaining, we will be letting waitlisted people in.</p>
```

Comment:
instead of `{{ event_start_date.strftime("%I%p") }}` it should use `WAITLISTED_ACCEPTANCE_START_TIME`